Makefile

```diff
@@ -0,0 +1,52 @@
+BIN     = $$(npm bin)
+BABEL   = $(BIN)/babel
+KARMA   = $(BIN)/karma
+WEBPACK = $(BIN)/webpack
+
+.PHONY: clean test test-watch build package.json javascript release example javascript-min
```

Comment:
.PHONY tells make to _always_ run a command even if the files listed (see the `javascript` task) have not updated. For now, I'm more or less phonying everything. I'd like to be smarter about this.